src/components/Articles.jsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+import { ArticlesList } from "./ArticlesList";
+import { TopicsSidebar } from "./TopicsSidebar";
+import { ArticleFilters } from "./ArticleFilters";
+import { getArticles, getTopics } from "../articles_api_utils";
+import { PageNavigator } from "./PageNavigator";
+
+export const Articles = () => {
+  const [articlesList, setArticlesList] = useState([]);
+  const [totalArticles, setTotalArticles] = useState(articlesList.length);
+  const [topics, setTopics] = useState([]);
+  const [topicsFilter, setTopicsFilter] = useState("All");
+  const [sort_by, setSort_By] = useState("");
+  const [order, setOrder] = useState("");
+  const [limit, setLimit] = useState("10");
+  const [p, setP] = useState("1");
+
+  useEffect(() => {
```

Comment:
Ideally we want to be adding some sort of loading state, so that if we have to wait for the data to come through from the API, we can display some sort of loading message or icon. Have a look to add this ✍️